contracts/libraries/helper/Errors.sol

```diff
@@ -12,25 +12,26 @@ library Errors {
     string public constant VL_ONLY_COUNTERPARTY = "2";
     string public constant VL_LONG_PRICE_THAN_CURRENT_PRICE = "3";
     string public constant VL_SHORT_PRICE_LESS_CURRENT_PRICE = "4";
-    string public constant VL_INVALID_SIZE = "6";
-    string public constant VL_INVALID_ORDER_ID = "7";
-    string public constant VL_EXPIRED = "8";
-    string public constant VL_NOT_ENOUGH_LIQUIDITY = "9";
-    string public constant VL_NOT_ENOUGH_QUOTE_FUNDING = "10";
-    string public constant VL_NOT_ENOUGH_BASE_FUNDING = "11";
-    string public constant VL_MUST_NOT_FILLED = "12";
-    string public constant VL_SPOT_MANGER_NOT_EXITS = "13";
-    string public constant VL_SPOT_MANGER_EXITS = "14";
-    string public constant VL_NO_AMOUNT_TO_CLAIM = "15";
-    string public constant VL_NO_LIMIT_TO_CANCEL = "16";
-    string public constant VL_ONLY_OWNER = "17";
-    string public constant VL_MUST_IDENTICAL_ADDRESSES = "18";
-    string public constant VL_MUST_NOT_INITIALIZABLE = "19";
-    string public constant VL_MUST_NOT_TOKEN_USE_RFI = "20";
+    string public constant VL_INVALID_SIZE = "6.1";
+    string public constant VL_INVALID_PAIR_INFO = "6.2";
+    string public constant VL_INVALID_ORDER_ID = "6.3";
+    string public constant VL_EXPIRED = "7";
+    string public constant VL_NOT_ENOUGH_LIQUIDITY = "8";
+    string public constant VL_NOT_ENOUGH_QUOTE_FUNDING = "9";
+    string public constant VL_NOT_ENOUGH_BASE_FUNDING = "10";
+    string public constant VL_MUST_NOT_FILLED = "11";
+    string public constant VL_SPOT_MANGER_NOT_EXITS = "12";
+    string public constant VL_SPOT_MANGER_EXITS = "13";
+    string public constant VL_NO_AMOUNT_TO_CLAIM = "14";
+    string public constant VL_NO_LIMIT_TO_CANCEL = "15";
```

Comment:
when you change code number of Errors, must notify to FE team to change the mapping code for texts error

Reply:
we can refactor error codes when all contracts done as it can be updated when developing